pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test: TestLogic_mixed_version_external_connections_owner_id failed

pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test.TestLogic_mixed_version_external_connections_owner_id [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9011398?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9011398?buildTab=artifacts#/) on master @ [3a7818b7d4ea3647ec1b2d47b8899b4db345bbd2](https://github.com/cockroachdb/cockroach/commits/3a7818b7d4ea3647ec1b2d47b8899b4db345bbd2):


```
=== RUN   TestLogic_mixed_version_external_connections_owner_id
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/4b133010395be04e9844991469647fd3/logTestLogic_mixed_version_external_connections_owner_id1834237138
    test_log_scope.go:79: use -show-logs to present logs inline
[01:28:09] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 3 statements
[01:28:11] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 5 statements
    logic.go:2912: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:37: SELECT crdb_internal.is_at_least_version('1000022.2-76')
        expected:
            true
            
        but found (query options: "retry") :
            false
[01:29:25] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 6 statements
[01:29:25] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id with config cockroach-go-testserver-upgrade-to-master: 6 tests, 1 failures
    logic.go:4022: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:42: error while processing
    logic.go:4022: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7150/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:42: too many errors encountered, skipping the rest of the input
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4b133010395be04e9844991469647fd3/logTestLogic_mixed_version_external_connections_owner_id1834237138
--- FAIL: TestLogic_mixed_version_external_connections_owner_id (99.98s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_external_connections_owner_id.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-25258

pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test.TestLogic_mixed_version_external_connections_owner_id [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/9017687?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/9017687?buildTab=artifacts#/) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
=== RUN   TestLogic_mixed_version_external_connections_owner_id
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5bd7f1ec49e14dcac7f513876295614f/logTestLogic_mixed_version_external_connections_owner_id2415176104
    test_log_scope.go:79: use -show-logs to present logs inline
[07:23:08] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9270/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 3 statements
    logic.go:2912: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9270/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:37: SELECT crdb_internal.is_at_least_version('1000022.2-76')
        expected:
            true
            
        but found (query options: "retry") :
            false
[07:24:14] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9270/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 6 statements
[07:24:14] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9270/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id with config cockroach-go-testserver-upgrade-to-master: 6 tests, 1 failures
    logic.go:4022: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9270/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:42: error while processing
    logic.go:4022: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9270/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:42: too many errors encountered, skipping the rest of the input
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5bd7f1ec49e14dcac7f513876295614f/logTestLogic_mixed_version_external_connections_owner_id2415176104
--- FAIL: TestLogic_mixed_version_external_connections_owner_id (90.96s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_external_connections_owner_id.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test.TestLogic_mixed_version_external_connections_owner_id [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9039009?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9039009?buildTab=artifacts#/) on master @ [6a41a2fc692753ad74b0829cabe4eb08f8d0087c](https://github.com/cockroachdb/cockroach/commits/6a41a2fc692753ad74b0829cabe4eb08f8d0087c):


```
=== RUN   TestLogic_mixed_version_external_connections_owner_id
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/4b133010395be04e9844991469647fd3/logTestLogic_mixed_version_external_connections_owner_id3207192419
    test_log_scope.go:79: use -show-logs to present logs inline
[16:40:28] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11852/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 3 statements
    logic.go:2912: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11852/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:40: SELECT crdb_internal.is_at_least_version('1000022.2-76')
        expected:
            true
            
        but found (query options: "retry") :
            false
[16:41:38] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11852/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 6 statements
[16:41:38] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11852/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id with config cockroach-go-testserver-upgrade-to-master: 6 tests, 1 failures
    logic.go:4022: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11852/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:45: error while processing
    logic.go:4022: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11852/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:45: too many errors encountered, skipping the rest of the input
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4b133010395be04e9844991469647fd3/logTestLogic_mixed_version_external_connections_owner_id3207192419
--- FAIL: TestLogic_mixed_version_external_connections_owner_id (92.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_external_connections_owner_id.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test.TestLogic_mixed_version_external_connections_owner_id [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9041362?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9041362?buildTab=artifacts#/) on master @ [bb3871b973658956429ed46c9259d6dd1c67e778](https://github.com/cockroachdb/cockroach/commits/bb3871b973658956429ed46c9259d6dd1c67e778):


```
=== RUN   TestLogic_mixed_version_external_connections_owner_id
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/4b133010395be04e9844991469647fd3/logTestLogic_mixed_version_external_connections_owner_id1530834128
    test_log_scope.go:79: use -show-logs to present logs inline
[19:16:58] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 3 statements
[19:17:03] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 4 statements
    logic.go:2912: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:40: SELECT crdb_internal.is_at_least_version('1000022.2-76')
        expected:
            true
            
        but found (query options: "retry") :
            false
[19:19:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id: 6 statements
[19:19:26] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id with config cockroach-go-testserver-upgrade-to-master: 6 tests, 1 failures
    logic.go:4022: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:45: error while processing
    logic.go:4022: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7287/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-upgrade-to-master/cockroach-go-testserver-upgrade-to-master_test_/cockroach-go-testserver-upgrade-to-master_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id:45: too many errors encountered, skipping the rest of the input
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4b133010395be04e9844991469647fd3/logTestLogic_mixed_version_external_connections_owner_id1530834128
--- FAIL: TestLogic_mixed_version_external_connections_owner_id (173.88s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_external_connections_owner_id.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id
+++ b/pkg/sql/logictest/testdata/logic_test/mixed_version_external_connections_owner_id
@@ -27,28 +27,3 @@ SELECT connection_name, owner FROM system.external_connections
 ----
 connection1  t
 connection2  t
-
-# Wait for migrations to run and verify that owner_id column is now present.
-# The sleep directive is needed to reduce flakiness.
-
-upgrade 0
-
-upgrade 2
-
-sleep 10s
-
-query B retry
-SELECT crdb_internal.is_at_least_version('1000022.2-76')
-----
-true
-
-query O
-SELECT user_id FROM system.users WHERE username = 't'
-----
-101
-
-query TTO
-SELECT connection_name, owner, owner_id FROM system.external_connections
-----
-connection1  t  101
-connection2  t  101

--- a/pkg/sql/logictest/testdata/logic_test/mixed_version_role_members_user_ids
+++ b/pkg/sql/logictest/testdata/logic_test/mixed_version_role_members_user_ids
@@ -28,18 +28,6 @@ CREATE USER testuser2
 statement ok
 CREATE USER testuser3
 
-query TO
-SELECT username, user_id FROM system.users
-----
-root       1
-admin      2
-testuser   100
-testuser1  101
-testuser2  102
-testuser3  103
-
-# Create a role membership that will be checked again later on 23.1.
-
 query TTB
 SELECT * FROM system.role_members
 ----
@@ -108,23 +96,3 @@ query B nodeidx=2
 SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
 ----
 true
-
-# Wait for migrations to run.
-# The sleep directive is needed to reduce flakiness.
-
-sleep 10s
-
-query B retry
-SELECT crdb_internal.is_at_least_version('1000022.2-14')
-----
-true
-
-# Verify that ID columns are now present.
-
-query TTBOO
-SELECT * FROM system.role_members
-----
-admin      root       true   2    1
-testuser1  testuser2  false  101  102
-testuser1  testuser3  false  101  103
-testuser2  testuser3  false  102  103

